public/app/features/alerting/unified/RuleEditorGrafanaRules.test.tsx

```diff
@@ -142,7 +196,7 @@ describe('RuleEditor grafana managed rules', () => {
 
     await userEvent.type(await ui.inputs.name.find(), 'my great new rule');
 
-    const folderInput = await ui.inputs.folder.find();
+    const folderInput = ui.inputs.folder.get();
```

Comment:
Any particular reason we changed this from `await find()`? I usually see these when a flakey test would otherwise be triggered.

Reply:
good catch, I forgot to restore this change after fixing the test